doc/whats_new.rst

```diff
@@ -230,6 +230,9 @@ Enhancements
      (`#6846 <https://github.com/scikit-learn/scikit-learn/pull/6846>`_)
      By `Sebastian Säger`_ and `YenChen Lin`_.
 
+   - Added new return type ``(data, target)`` : tuple option to :func:`load_iris` dataset. (`#7049 <https://github.com/scikit-learn/scikit-learn/pull/7049>`_) 
+     By `Manvendra Singh`_ and `Nelson Liu`_.   
```

Comment:
Your code review resulted in major changes in the initial PR . I think you had a decent part to play in this contribution. :smile: